examples/utils.py

```diff
@@ -118,3 +117,8 @@
     ('LogPlotTest', 'LogPlotTest.py'),  # Plotting.py
     ('ViewLimits', 'ViewLimits.py'),    # ViewBoxFeatures.py
 ])
+
+# examples that are not suitable for CI testing
+skiptest = dict([
+    ('ProgressDialog', 'ProgressDialog.py'),    # modal dialog
```

Comment:
Why don't we skip when `os.getenv("CI")`, since that variable is `True` on CI platforms, that way we can still run this test locally.

Reply:
I am not so sure about this. ProgressDialog.py is different from the rest in that it doesn't run the event loop via app.exec_().
Meanwhile, test_examples.py assumes that an event loop needs to run and does it by calling processEvents.

Reply:
Ahhh!  That is a very relevant distinction; thanks for catching it.